internal/xds/translator/testdata/out/xds-ir/address-as-listener-name.routes.yaml

```diff
@@ -0,0 +1,70 @@
+- ignorePortInHostMatching: true
+  name: envoy-gateway/gateway-1/http
```

Comment:
Do we also want to rename the RouteConfig and VirtualHost using the same address pattern? Like the listener name, their current name follows one of the random Gateway Listener when multiple listeners are on the same port.

Reply:
is this 1:1 with listener ? I thought vhost was 1:1 with hostname

Reply:
RouteConfig is 1:1 listener, VirtualHost name is prefixed with listener name:

Examples:

Listener:  default/gateway-1/http   (now tcp-80)
RouteConfig:   default/gateway-1/http
VirtualHost:  default/gateway-1/http/www_foo_com